__tests__/components/Select.spec.tsx

```diff
@@ -21,14 +21,34 @@ describe("Form", () => {
     expect(!!getByText("Tech")).toBe(true);
   });
 
+  it("should display placeholder", () => {
+    const { getByText } = render(
+      <Form onSubmit={jest.fn()}>
+        <Select
+          placeholder="Select..."
+          options={[{ id: "node", title: "NodeJS" }]}
+          name="tech"
+          label="Tech"
+        />
+      </Form>
+    );
+
+    expect(!!getByText("Select...")).toBe(true);
+  });
+
   it("should display error", async () => {
     const schema = Yup.object().shape({
       tech: Yup.string().required("Tech is required")
     });
 
+    const initialData = {
```

Comment:
Why did you changed other test? I mean, if the initial data is not present with empty value, so the placeholder is being returned as value? This should not occurs.

Reply:
if we set a placeholder and not provide a default value to the *`Select` component*, the *`select` html element* don't knows what `option` element to choose as his value. in order to solve this, we need to provide a default value to the *`Select` component*.

Reply:
lol. reading my previous comment i can see that it's smells like a bug :bug: 

i solve this providing a default placeholder(an empty option) to be used when the developer not set one. if no default value is defined the placeholder will be selected. 

and the changed tests are rollbacked(this term exists :laughing:).

thanks for your feedback @diego3g. review it again and tell me what you think about. any other feedback is always welcome (: